content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 50
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on and use
+the web-based graphical user interface for viewing service graphs of the mesh
+as well as your Istio configuration objects such as virtual services
+and destination rules. You will also use the Kiali API to generate graph data
+in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as
+the example application throughout this task.
+
+## Before you begin
+
+* [Install Istio](/docs/setup/) in your cluster and deploy an
```

Comment:
The docs/setup don't show how to enable Kiali and Kiali atm is disabled by default.  Can you change the docs or the state.  IIRC from the last TOC meeting was to keep kiali disabled until there was some basic smoke testing in the gate to make sure it comes up properly.

Reply:
Yes, good point. It will be disabled by default (except for the demo yamls, which I believe we agreed to have everything plus the kitchen sink enabled in those :))

I will add some blurb here explaining how to at least get it to be installed via the helm chart (involves creating a secret and passing `--set kiali.enabled=true` to helm as described in https://github.com/morvencao/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart)

Reply:
I tried addressing some of this in the commit: https://github.com/istio/istio.io/pull/2715/commits/271fa9980e2e679563157d538565656549f4c7e9